### DISCUSSION TITLE
This library works better than the Adafruit one

### DISCUSSION THREAD

**Discussion by yucandu** [wollewald/ADS1115_WE#34] [General]
Just wanted to compliment you on your library.  I discovered it when trying to build an ultra low power battery powered device.  The Adafruit_ADS1X15 library doesn't allow the ADS1115 to go to sleep for some reason.  Identical hardware, identical setup, but switching from the single shot example in their library to the single shot example in your library, and deep sleep current is reduced by 300uA, consistently.

**Comment by wollewald** [comment]
@moeburn thank you for your kind comment!
